sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/MainActivity.java

```diff
@@ -146,6 +147,13 @@ protected void onCreate(Bundle savedInstanceState) {
           }
         });
 
+    binding.openSecondActivity.setOnClickListener(
+        view -> {
+          // finishing so its completely destroyed
+          finish();
```

Comment:
Is this needed? As a developer reading this code, should I be using this pattern on my app?

Reply:
yes, I wanted a cold start